package.json

```diff
@@ -21,6 +21,7 @@
     "bcryptjs": "^2.4.3",
     "cookie-parser": "^1.4.6",
     "cors": "^2.8.5",
+    "date-fns": "^2.30.0",
```

Comment:
Did this come from merging `main` or something you added?

Reply:
@akosasante oh i will delete it, i added this package because I was not sure how to add a Date() property- didn't know that it should be a string